pkg/oc/cli/cancelbuild/cancelbuild.go

```diff
@@ -200,19 +209,25 @@ func (o *CancelBuildOptions) RunCancelBuild() error {
 				break
 			}
 		}
-		if stateMatch && !buildutil.IsBuildComplete(build) {
+
+		internalBuildStatus := &buildapi.BuildStatus{}
+		if err := oapiv1.Convert_v1_BuildStatus_To_build_BuildStatus(&build.Status, internalBuildStatus, nil); err != nil {
+			return err
+		}
```

Comment:
I am converting the build status to internal here since the buildutil helper only deals with internals